index.html

```diff
@@ -1,89 +1,84 @@
+            '/': 'Search'
+          }
+        },
+        pagination: {
+          crossChapter: true
         }
-      },
-      pagination: {
-        crossChapter: true
-      }
-    };
-  </script>
-  <!-- <script src="//unpkg.com/docsify/lib/docsify.min.js"></script> -->
-  <script src="./resource/js/docsify.min.js"></script>
-  <!-- <script src="//unpkg.com/docsify-pagination@2.4.0/dist/docsify-pagination.min.js"></script> -->
-  <script src="./resource/js/docsify-pagination.min.js"></script>
-  <!-- <script src="//unpkg.com/docsify-copy-code@2"></script> -->
-  <script src="./resource/js/docsify-code-code.js"></script>
-  <!-- <script src="https://cdn.jsdelivr.net/npm/docsify-tabs@1"></script> -->
-  <script src="./resource/js/docsify-tabs.js"></script>
-  <!-- <script src="//unpkg.com/docsify/lib/plugins/search.min.js"></script> -->
-  <script src="./resource/js/search.min.js"></script>
-  <script src="//unpkg.com/prismjs/components/prism-rust.min.js"></script>
-  <!-- <script src="//unpkg.com/prismjs/components/prism-bash.min.js"></script> -->
-  <script src="./resource/js/prism-bash.min.js"></script>
-  <!-- Custom JS for CodeView -->
-  <!-- <script defer type="text/javascript" src="//unpkg.com/monaco-editor@0.15.6/min/vs/loader.js"></script> -->
-  <script defer type="text/javascript" src="./resource/js/loader.js"></script>
-  <script defer src="./custom/codeview.js"></script>
-  <!-- Gitter Chat -->
-  <script>
-    ((window.gitter = {}).chat = {}).options = {
-      room: 'libp2p-enthusiasts/community',
-      activationElement: true
-    };
-  </script>
-  <!-- <script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script> -->
-  <script src="./resource/js/sidecar.v1.js" async defer></script>
-</body>
-
+      };
+    </script>
+    <!-- <script src="//unpkg.com/docsify/lib/docsify.min.js"></script> -->
+    <script src="./resource/js/docsify.min.js"></script>
+    <!-- <script src="//unpkg.com/docsify-pagination@2.4.0/dist/docsify-pagination.min.js"></script> -->
+    <script src="./resource/js/docsify-pagination.min.js"></script>
+    <!-- <script src="//unpkg.com/docsify-copy-code@2"></script> -->
+    <script src="./resource/js/docsify-code-code.js"></script>
+    <!-- <script src="https://cdn.jsdelivr.net/npm/docsify-tabs@1"></script> -->
+    <script src="./resource/js/docsify-tabs.js"></script>
+    <!-- <script src="//unpkg.com/docsify/lib/plugins/search.min.js"></script> -->
+    <script src="./resource/js/search.min.js"></script>
+    <script src="//unpkg.com/prismjs/components/prism-rust.min.js"></script>
+    <!-- <script src="//unpkg.com/prismjs/components/prism-bash.min.js"></script> -->
+    <script src="./resource/js/prism-bash.min.js"></script>
+    <!-- Custom JS for CodeView -->
+    <!-- <script defer type="text/javascript" src="//unpkg.com/monaco-editor@0.15.6/min/vs/loader.js"></script> -->
+    <script defer type="text/javascript" src="./resource/js/loader.js"></script>
```

Comment:
@luke-rucker I don't remember why I did this. It could be just for some testing and then forgot to fix it. I think we should correct this.

Reply:
@shresthagrawal Done!